dlrover/python/master/scaler/pod_scaler.py

```diff
@@ -162,14 +163,44 @@ def _retry_to_get_job(self):
         return None
 
     def _retry_to_get_master_pod(self):
-        master_name = f"elasticjob-{self._job_name}-dlrover-master"
+        """
+        Get master pod by labels.
+        Notice: Labels might be different in different environments for now.
+        """
+        master_labels = {
```

Comment:
Use a method to generate the labels of master pod. Then, we can add a test case for the method.